canvas_modules/common-canvas/src/common-properties/controls/control-factory.js

```diff
@@ -332,9 +336,14 @@ export default class ControlFactory {
 			const className = control.className ? control.className : "";
 			return (
 				<div key={"properties-ctrl-" + control.name} data-id={"properties-ctrl-" + control.name}
-					className={classNames("properties-ctrl-wrapper", { "hide": hidden }, className)}
+					className={classNames(
+						"properties-ctrl-wrapper",
+						{ "hide": hidden, "action": !isNil(action) && has(control, "action.image.placement") },
```

Comment:
Do you need to add this check `!isNil(action)`?  I would think the `has` check would be enough.